src/ai/backend/agent/agent.py

```diff
@@ -858,6 +860,16 @@ async def _pipeline(r: Redis):
                 )
             )
 
+        _container_observer_task = HeartbeatTask(
+            self.id,
+            self.enumerate_containers,
+            self.event_producer,
+        )
```

Comment:
It doesn't seem necessary to necessarily attach a `_` as a prefix to variables in a function.